<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="10686" started-at="2019-01-24T10:23:56Z" finished-at="2019-01-24T10:24:07Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="10686" started-at="2019-01-24T10:23:56Z" finished-at="2019-01-24T10:24:07Z">
      <class name="com.atmecs.automation.FirstTestNg.LearnTestNg">
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:com.atmecs.automation.FirstTestNg.LearnTestNg@46d56d67]" name="beforeMethod" is-config="true" duration-ms="2451" started-at="2019-01-24T10:23:56Z" finished-at="2019-01-24T10:23:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="testCurrentUrl()[pri:0, instance:com.atmecs.automation.FirstTestNg.LearnTestNg@46d56d67]" name="testCurrentUrl" duration-ms="47" started-at="2019-01-24T10:23:58Z" finished-at="2019-01-24T10:23:59Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [https://www.google.com/] but found [data:,]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [https://www.google.com/] but found [data:,]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.atmecs.automation.FirstTestNg.LearnTestNg.testCurrentUrl(LearnTestNg.java:40)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCurrentUrl -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.atmecs.automation.FirstTestNg.LearnTestNg@46d56d67]" name="afterMethod" is-config="true" duration-ms="876" started-at="2019-01-24T10:23:59Z" finished-at="2019-01-24T10:23:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:com.atmecs.automation.FirstTestNg.LearnTestNg@46d56d67]" name="beforeMethod" is-config="true" duration-ms="1189" started-at="2019-01-24T10:23:59Z" finished-at="2019-01-24T10:24:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="PASS" signature="testTitle()[pri:0, instance:com.atmecs.automation.FirstTestNg.LearnTestNg@46d56d67]" name="testTitle" duration-ms="5496" started-at="2019-01-24T10:24:01Z" finished-at="2019-01-24T10:24:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testTitle -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.atmecs.automation.FirstTestNg.LearnTestNg@46d56d67]" name="afterMethod" is-config="true" duration-ms="605" started-at="2019-01-24T10:24:06Z" finished-at="2019-01-24T10:24:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- com.atmecs.automation.FirstTestNg.LearnTestNg -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
